### PR TITLE
Disallow installation of Sf FrameworkBundle 5.2.6

### PR DESCRIPTION
**Pull request by aschempp**
see https://github.com/symfony/symfony/issues/40618
Should be set to an exact version once this is fixed in Symfony

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
         "knplabs/knp-time-bundle": "1.9.0",
         "lexik/maintenance-bundle": "2.1.4",
         "symfony/finder": "3.4.7 || 4.0.7",
-        "symfony/framework-bundle": "4.2.7",
+        "symfony/framework-bundle": "4.2.7 || > 5.2.5",
```
Comment (leofeyer) [inline composer.json:20]:
```suggestion
        "symfony/framework-bundle": "4.2.7 || 5.2.5",
```

AFAIR, we never use unbound version constraints for conflicts. If the issue is not fixed in version 5.2.6, we would update this file again.

Comment (aschempp) [reply composer.json:20]: I think we do?
https://github.com/contao/conflicts/commit/8df62d28c5e2dd67380d1503e490285d2ef24362
https://github.com/contao/conflicts/commit/8166d09b3b0ea81f71e4c0af7b533eac5104902a

**Review by m-vo — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (leofeyer) [reply composer.json:20]: Unbound version constraints bear the risk of being forgotten and suddenly people do not get updates of the package anymore without even noticing. I though that was the reasoning behind not using unbound version constraints.

But let's wait what @ausi says.

Comment (aschempp) [reply composer.json:20]: We'll I'm following https://github.com/symfony/symfony/issues/40618 and we will certainly change this as soon as another fix is available 🤷

Comment (ausi) [reply composer.json:20]: I’m OK with the open version constraint. We should IMO create an issue to change that, and mark that issue as blocked by symfony/symfony#40618

**Review by ausi — APPROVED** [review]

**Pull request merged**